kits/daisy/src/xml_util/xml_helpers.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "pugixml.hpp"
+
+namespace hebi {
+
+class XMLHelpers
```

Comment:
Why is this here? Why aren't they free standing functions?

Reply:
If you really need them to be scoped, there's no reason to just place them in a namespace like `hebi::xml`. Putting them in a class where there are only `static` functions is confusing

Reply:
I think this came from an ugly period in my life where I coded in Java.  I profusely apologize.